lib/bash/common.bash

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+# Source this file to take advantage of functions.
```

Comment:
This script looks good to me, but how about putting it a different location, somewhere under `scripts`? Perhaps `scripts/lib/common.bash`?

Reply:
👍